.travis.yml

```diff
@@ -24,7 +24,8 @@ script:
   - ./contrib/get_kind.sh
   - ./contrib/create_kind_cluster.sh
   - ./contrib/deploy.sh
-  - ./contrib/install_tiller.sh
```

Comment:
Where did install_tiller.sh go?

Reply:
I am still learning a bit of kubernetes, I've pushed update to install tiller. My reasoning was that it might not add much value to the test and I could remove it to save time on testing, though I've added it now.